docs/jazzy.yml

```diff
@@ -112,3 +125,30 @@ custom_categories:
       - RouteSnappingMinimumHorizontalAccuracy
       - RouteSnappingMinimumSpeed
       - NavigationSettings
+  - name: User Feedback collection
+    children:
+      - FeedbackViewController
+      - FeedbackViewControllerDelegate
+      - FeedbackItem
+      - FeedbackType
+      - IncorrectVisualSubtype
+      - ConfusingAudioSubtype
+      - RouteQualitySubtype
+      - IllegalRouteSubtype
+      - RoadClosureSubtype
+      - FeedbackSource
+      - EndOfRouteFeedback
+  - name: MGL Additions
```

Comment:
These are all extensions to map SDK classes, so we can fold them into the “Maps” category.